learning/katas/java/util/src/org/apache/beam/learning/katas/util/Log.java

```diff
@@ -70,6 +70,7 @@ public void processElement(
                   }
 
                   LOG.info(message);
+                  System.out.println(message)
```

Comment:
Should we instead raise the severity of the logging, as this line seems redundant with the above?

Reply:
@robertwb as @pabloem mentioned, it is done for Beam Playground, which shows both logs and stdout for the executed code.

@pabloem I see that this way we will have in Beam Playground almost the same output in both tabs, are we okay with this?

Reply:
sorry about the delay on this - if we increase the severity of logging, we still don't get the output on Stdout, right?

Reply:
Generally logging only shows up on stderr. (Does the playground not show stderr?) 

Given this transform is explicitly about logging, it feels really odd to also print to stdout. We could decide to use a stdout-printing transform rather than a logging transform for the katas.

Reply:
playground shows both stderr and stdout, but stderr contains a lot more output (all logging from the pipeline run, after all)

I think I like your suggestion about having a print-specific transform. Let me see if this works fine